listenbrainz/webserver/views/stats_api.py

```diff
@@ -0,0 +1,57 @@
+from datetime import datetime
+from flask import Blueprint, request, jsonify, current_app
+from listenbrainz.webserver.errors import APIBadRequest, APIInternalServerError,\
+    APIUnauthorized, APINotFound, APIServiceUnavailable
+from listenbrainz.webserver.decorators import crossdomain
+import listenbrainz.db.user as db_user
+import listenbrainz.db.stats as db_stats
+from listenbrainz.webserver.rate_limiter import ratelimit
+
+stats_api_bp = Blueprint('stats_api_v1', __name__)
+
+
+@stats_api_bp.route("<user_name>/artist")
+@crossdomain()
+@ratelimit()
+def get_artist(user_name):
+    """
+    Get top artists for user ``user_name``. The format for the JSON returned is defined in our :ref:`json-doc`.
+
+    **Note**: This endpoint is currently in beta
+
+    :param count: Optional, number of artists to return.
+    :statuscode 200: Successfull query
+    :statuscode 204: Statistics for user haven't been calculated, empty response will be returned
+    :statuscode 400: Bad request, `count` should be of type integer
+    :statuscode 404: User not found
+    :resheader Content-Type: *application/json*
+    """
+
+    count = request.args.get('count', default=-1)
+    try:
+        count = int(count)
+    except ValueError:
+        raise APIBadRequest("Bad request, 'count' should be an integer")
```

Comment:
we should check that count is non-negative too, should raise a 400 if someone sends us -1